### PR TITLE
Updated schema sql : Caleb

### PR DESCRIPTION
**Pull request by CalebMathers**
For issue #100 

Amended schema sql to include the changes to image table, as well as changing body_assignment table to have "at" rather than begin and end times.

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -110,3 +102,15 @@ CREATE TABLE subscriber_county_assignment (
...
+    image_id BIGINT GENERATED ALWAYS AS IDENTITY,
+    image_name VARCHAR(10) NOT NULL,
+    image_url VARCHAR(255) NOT NULL,
+    image_date DATE NOT NULL,
+    region_id SMALLINT,
+    constellation_id SMALLINT,
+    PRIMARY KEY (image_id),
+    FOREIGN KEY (region_id) REFERENCES region(region_id),
+    FOREIGN KEY (constellation_id) REFERENCES constellation(constellation_id)
+);
```
Comment (AlexOBarnes) [inline database/schema.sql:116]: This makes it a lot easier to sort by region and constellation

File: database/schema.sql [hunk 2]
```diff
@@ -35,6 +27,7 @@ CREATE TABLE region (
 CREATE TABLE constellation (
     constellation_id SMALLINT GENERATED ALWAYS AS IDENTITY,
     constellation_name VARCHAR(35) NOT NULL,
```
Comment (AlexOBarnes) [inline database/schema.sql:29]: Made add unique constraints to all name variables in the ERD?
```suggestion
    constellation_name VARCHAR(35) NOT NULL UNIQUE,
```

**Review by AlexOBarnes — CHANGES_REQUESTED** [review]
Maybe consider more stringent constraints
Good changes to image table

**Review by AlexOBarnes — APPROVED** [review]
Thanks for making these changes

**Comment by AlexOBarnes** [comment]
This schema is a lot more robust to errors now which is great

**Review by olliejth — APPROVED** [review]
Changes made correctly as far as I can see

File: database/schema.sql [hunk 3]
```diff
@@ -110,3 +102,15 @@ CREATE TABLE subscriber_county_assignment (
     FOREIGN KEY (subscriber_id) REFERENCES subscriber(subscriber_id),
     FOREIGN KEY (county_id) REFERENCES county(county_id)
 );
+
```
Comment (MurasatoNaoya) [inline database/schema.sql:105]: LGTM, needed this inclusion to account for NASA image-of-the-day, moon phase images etc..

**Review by MurasatoNaoya — APPROVED** [review]
Solid update to the schema constructing script, now fully accounting for all tables in the ERD.

**Pull request merged**